sentry-core/Cargo.toml

```diff
@@ -26,17 +26,16 @@ client = ["rand"]
 # and macros actually expand features (and extern crate) where they are used!
 debug-logs = ["dep:log"]
 test = ["client"]
-UNSTABLE_metrics = ["sentry-types/UNSTABLE_metrics", "regex"]
-UNSTABLE_cadence = ["dep:cadence", "UNSTABLE_metrics"]
+metrics = ["sentry-types/metrics", "regex", "crc32fast"]
+metrics-cadence1 = ["dep:cadence", "metrics"]
```

Comment:
Out of interest, why is it `cadence1`?

Reply:
probably not too relevant for the SDK because we have semver-breaking changes all the time, but the idea was to be able to support multiple versions of cadence without having to do a semver-breaking bump on our end.